delta/plugins/composite-views/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/compositeviews/client/DeltaClient.scala

```diff
@@ -48,8 +48,19 @@ final class DeltaClient(client: HttpClient, retryDelay: FiniteDuration)(implicit
     client.fromJsonTo[ProjectCount](statisticsEndpoint)
   }
 
-  private def toOffset(id: String): Offset =
-    Try(TimeBasedUUID(UUID.fromString(id))).orElse(Try(Sequence(id.toLong))).getOrElse(NoOffset)
+  /**
+    * Checks whether the events endpoint and token provided by the source are correct
+    *
+    * @param source the source
+    */
+  def checkEvents(source: RemoteProjectSource): HttpResult[Unit] = {
+    val uri                              =
+      source.endpoint / "resources" / source.project.organization.value / source.project.project.value / "events"
+    implicit val cred: Option[AuthToken] = token(source)
+    client(Head(uri).withCredentials) {
+      case resp if resp.status.isSuccess() => UIO.delay(resp.discardEntityBytes()) >> IO.unit
```

Comment:
and if it's not successful?

Reply:
The it with go on the left raiseerror channel of it IO

Reply:
will the entity bytes be discarded?

Reply:
Yes, [that's the case](https://github.com/BlueBrain/nexus/blob/master/delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/http/HttpClient.scala#L101).